locale/ar/site.json

```diff
@@ -165,6 +165,11 @@
     "index": {
       "text": "إصدارات الأمانة الجديدة متوفرة الآن لإصدارات 14.x و 12.x."
     },
+    "saveukraine": {
+      "visible": true,
+      "text": "#CloseTheSky over #Ukraine",
+      "link": "https://twitter.com/ua_parliament/status/1498266783173824517"
```

Comment:
Given that it is possible (and more common) for individual tweets to be deleted, This link should likely go to the profile instead (https://twitter.com/ua_parliament).

Reply:
I believe this tweet can show what situation it is, and what kind of help we need. Adding just a link to profile will be boring and useful for anyone.